content/soarian/dstu2/diagnostic/diagnostic-report.md

```diff
@@ -49,11 +49,11 @@ _Implementation Notes_
 Notes:
 
 * See details regarding special [date](../../#special-information-regarding-date-parameters) requirements.
-* If the query date parameter is null in the query, the API will return qualifying records with dates less than or equal to the date of query as limited by the lesser of 1000 records or 30 days inclusive of the query date.
-* If a single date parameter is included, the response will include qualifying records dated that day as limited by 1000 records.
-* If a valid date range is used in the query, the API will return qualifying records within the dates specified as limited by the lesser of 1000 records or 30 days starting at the upper limit date specified.
-* If a date range is used greater than 30 days, the response will include qualifying records within the dates specified as limited by the lesser of 1000 records or 30 days starting at the upper limit date specified and include in the informational response in the narrative extension that the date range entered is greater than 30 days and please refine your response.
-* If an invalid date range is specified in the query, the API will error.
+* If the query date parameter is null in the query, the API will return all qualifying records.
+* If a single date parameter is included, the response will include all qualifying records dated that day.
+* If a valid date range is used in the query, the API will return all qualifying records within the dates specified.
+* If an invalid date range is used in the query, the API will error with code [500] [common-errors]. Please refer to [Special information regarding date parameters] [date-parameter].
```

Comment:
These links don't work - you need to add them at the bottom of the file just like CarePlan